staff/joseluis-juste/app-template/home.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>ABOUT</title>
+</head>
+
+<body>
+    <header>
+        <img src="https://picsum.photos/200/200/?" alt="">
+    </header>
+    <nav>
+        <a href="about.html">About</a>
+        <a href="contact.html">Contact</a>
+        <a href="login.html">Login</a>
+        <a href="news.html">News</a>
+        <a href="profile.html">Profile</a>
+        <a href="home.html">Home</a>
+        <a href="register.html">Register</a>
+    </nav>
+
+    <aside>
+        <ul>
```

Comment:
remove repeated navigation links from here. html content should be right and not confusing.

Reply:
Ok, It has been fixed